index.js

```diff
@@ -86,8 +88,63 @@ module.exports = postcss.plugin('postcss-bem-linter', function(primaryOptions, s
       });
     }
 
+    function checkGlob(file, globs) {
```

Comment:
I couldn't use globby because globby actually checks the file system. `postcss-bem-linter` doesn't access the file system otherwise, let's leave file system access to the caller. I did clean it up to reduce the amount of custom logic.